Sources/MapboxMaps/Annotations/AnnotationOrchestrator.swift

```diff
@@ -57,6 +57,10 @@ public class AnnotationOrchestrator {
         self.displayLinkCoordinator = displayLinkCoordinator
     }
 
+    /// Dictionary of annotation managers keyed by their identifiers.
+    /// - NOTE: Removing an annotation manager from this collection can result in the corresponding annotations to be removed from the map view.
+    public var annotationManagersMap: [String: AnnotationManager] = [:]
```

Comment:
Let's name this annotationManagersById

Reply:
And can we make the setter private?

Reply:
And add a "remove annotation manager" or something like that

Reply:
https://github.com/mapbox/mapbox-maps-android/blob/main/plugin-annotation/src/main/java/com/mapbox/maps/plugin/annotation/AnnotationPluginImpl.kt#L63-L68

Something like that

Reply:
Agreed, addressed.